sections/conclusion.tex

```diff
@@ -0,0 +1,35 @@
+\section{Conclusion}\label{sec:conclusion}
+In this paper, we explore possibilities for incorporating metadata\vejleder{gerne ??} into existing topic models, such as \gls{lda} and \gls{pam}.
+We evaluate these models based on three different metadata: Author, Category, and Taxonomy, each of which represents the data in a different way.
+
+From the topic coherence results, shown in \autoref{tab:metric_results}, the \gls{pam} model using the Taxonomy metadata gets the best results.
+The Author and Category models based on \gls{lda} are the worst-performing, where the topic coherence is much lower than the other models.\vejleder{uddyb gerne denne paragraf}
+
+We want to answer the problem statement, which we stated in \autoref{sec:introduction}:
+\vejleder{lidt rodet måske.}
+
+\begin{itemize}
+	\item \textit{How does including metadata within the \gls{lda} model impact the resulting topics?}
+	\item \textit{What possible problems can these models help alleviate at Nordjyske?}
+	\begin{itemize}
+		\item \textit{More specifically, how can these models improve recommendation?}
+	\end{itemize}
+\end{itemize}
+
+We incorporate metadata into our topic models in multiple ways.
+The \gls{lda} model is used as a baseline, which indicates whether incorporating metadata can improve the topic quality of our topic models.
+Based on the results of Author-Topic and Category-Topic, we see that only using the metadata for topic assignment within \gls{lda} can hurt the topic quality.
+Other studies have shown that only using the metadata for topic assignment can improve the performance.
+This might be due to the particularities of the dataset, such as authors not usually writing about the same subject within the news environment.  

```

Comment:
```suggestion
This might be due to the particularities of our dataset, such as authors not usually writing about the same subject within the news environment.  
```